Documentation/ceph-disaster-recovery.md

```diff
@@ -288,7 +288,11 @@ ownerReferences:
 ```
 
 4. **After confirming all critical resources have had the owner reference to the CephCluster CR removed**, now
-   we allow the cluster CR to be deleted. Remove the finalizer by editing the CephCluster CR.
+   we allow the cluster CR to be deleted. 
```

Comment:
After reading this more thoroughly, I don't believe we need an update to the disaster recovery guide. This is really only about removing the finalizer from the CephCluster CR, and they don't need to remove the finalizer from the other resources. So the guide is fine without changes.

Reply:
`Restoring CRDs After Deletion`

The topic is regarding restoring the CRD, so it needs deletion and then a recreation of Cluster Resources, I think before deleting/clearing the CRDS we need to remove these finalizers too.

I am not sure without removing the other finalizer the CephCluster can be deleted.

PS: removed for now.

Reply:
This section of the disaster recovery is not about trying to clean up all the resources, but just change the cephcluster from deleted state to a restored state. It isn't affected by the other resources' finalizers.

Reply:
Okay 👍🏻